nibabel/gifti/gifti.py

```diff
@@ -375,6 +390,9 @@ def __init__(self,
         self.ext_offset = ext_offset
         self.dims = [] if self.data is None else list(self.data.shape)
 
+    def __repr__(self):
+        return f"<GiftiDataArray {intent_codes.label[self.intent]}{self.dims}>"
```

Comment:
Not knowing how this would work precisely, do you maybe need a space?
```suggestion
        return f"<GiftiDataArray {intent_codes.label[self.intent]} {self.dims}>"
```

Reply:
This currently shows as: `<GiftiDataArray pointset[32492, 3]>`. Your change would make it `<GiftiDataArray pointset [32492, 3]>`. I don't have a strong opinion on the aesthetics here.

Reply:
This is just me being nitpicky, I think. I also have no strong opinion but I know spaces are easy to miss in the f-strings when they get crowded like that.